pages/Documents/RichMessaging/AppleMessagesForBusinessTemplates/form-message-template.md

```diff
@@ -0,0 +1,349 @@
+
+  For all possible options for page configuration check [Apple Documentation](https://register.apple.com/resources/messages/msp-rest-api/type-interactive#form-message).
```

Comment:
Please unindent.

Reply:
what do you mean by unindent, i not sure what needs to be done.